cli/template/base/src/env.mjs

```diff
@@ -1,3 +1,5 @@
+// @ts-check
```

Comment:
No need for these as we use `checkJs` in tsconfig

Reply:
Yeah, I was thinking the same, but it wasn't working on VSCode for some reason. it's working now after restarting the instance.
Thx for pointing that out!